test/unit/application_validator_test.rb

```diff
@@ -14,72 +12,66 @@
 
 module Packwerk
   class ApplicationValidatorTest < Minitest::Test
+    include RailsApplicationFixtureHelper
+
     setup do
-      @configuration = Packwerk::Configuration.from_path("test/fixtures/skeleton")
+      setup_application_fixture
+    end
+
+    teardown do
+      teardown_application_fixture
     end
 
     test "validity" do
-      application_validator = Packwerk::ApplicationValidator.new(
-        config_file_path: @configuration.config_path,
-        configuration: @configuration
-      )
-      result = application_validator.check_all
+      use_template(:skeleton)
+
+      result = validator.check_all
 
       assert result.ok?, result.error_value
     end
 
     test "check_package_manifests_for_privacy returns an error for unresolvable privatized constants" do
-      application_validator = Packwerk::ApplicationValidator.new(
-        config_file_path: @configuration.config_path,
-        configuration: @configuration
-      )
+      use_template(:skeleton)
       ConstantResolver.expects(:new).returns(stub("resolver", resolve: nil))
 
-      result = application_validator.check_package_manifests_for_privacy
+      result = validator.check_package_manifests_for_privacy
 
       refute result.ok?, result.error_value
     end
 
     test "check_inflection_file returns error for mismatched inflections.yml file" do
-      config_path = "test/fixtures/skeleton/packwerk.yml"
-      configs = YAML.load_file(config_path)
-      configs["inflections_file"] = "different_inflections.yml"
-
-      configuration = Packwerk::Configuration.new(configs, config_path: config_path)
+      use_template(:skeleton)
+      merge_into_app_yaml_file("different_inflections.yml", { "acronym" => %w(TLA WTF LOL) })
```

Comment:
We can simply generate this yaml file for the one test that requires it.  No need to commit it to source control.